packages/std/src/results/cosmos_msg.rs

```diff
@@ -291,6 +294,8 @@ pub enum WasmMsg {
 ///     }))
 /// }
 /// ```
+/// Message types for interacting with the governance module.
+/// Allows contracts to cast votes and participate in governance processes.
```

Comment:
Same here. This should become the first paragraph of the doc-comment, not added at the end. I would also keep the link to the governance module.